tendermint-rs/Cargo.toml

```diff
@@ -35,8 +36,8 @@ failure_derive = "0.1"
 hkdf = { version = "0.7", optional = true }
 prost-amino = { version = "0.4.0", optional = true }
 prost-amino-derive = { version = "0.4.0", optional = true }
-rand = { version = "0.6", optional = true }
-ring = { version = "0.13", optional = true }
+rand_os = { version = "0.1", optional = true }
```

Comment:
We should probably use `rand_os` instead of `rand` from now on, at least anywhere we want a cryptographically secure RNG.

There's also the upcoming `getrandom` crate, which `rand_os` will eventually use as a dependency.